<!doctype html>
<html class="no-js" lang="en-us">
  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    <meta charset="utf-8">
    <title>Class 5: Genomics in the Digital Age | AI and Humanity</title>
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
    <link rel="stylesheet" href="http://localhost:1313/css/foundation.min.css">
    <link rel="stylesheet" href="http://localhost:1313/css/highlight.min.css">
    <link rel="stylesheet" href="http://localhost:1313/css/font-awesome.min.css">
    <link rel="stylesheet" href="http://localhost:1313/css/academicons.min.css">
    <link rel="stylesheet" href="http://localhost:1313/css/fonts.css">
    <link rel="stylesheet" href="http://localhost:1313/css/finite.css">
    <link rel="shortcut icon" href="/images/humanitylogo.png">  
    
  </head>
  <body>
      
    <header>
      <nav class="nav-bar">
	
	  <div class="title-bar" data-responsive-toggle="site-menu" data-hide-for="medium">	      
	    <button class="site-hamburger" type="button" data-toggle>
	      <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
	    </button>
	    <div class="title-bar-title site-title">
	      <a href="http://localhost:1313/">AI and Humanity</a>
	    </div>
	    <div class="title-bar-right pull-right">
	      


	    </div>
	  </div>
	    
	  
	    <div class="top-bar" id="site-menu" >	      
	      <div class="top-bar-title show-for-medium site-title">
		<a href="http://localhost:1313/">AI and Humanity</a>
	      </div>
	      <div class="top-bar-left">
		<ul class="menu vertical medium-horizontal">
		  
		  
		</ul>
	      </div>
	      <div class="top-bar-right show-for-medium">
		


	      </div>
	    </div>
	  
	</nav>
      
    </header>
    
    <main>
      

<div class="content">
    <article class="article" itemscope itemtype="http://schema.org/Article">
      <h1 itemprop="name" class="pagetitle">Class 5: Genomics in the Digital Age</h1>
      <div class="post-body" itemprop="articleBody">
        <p><strong>Blogging Team 11</strong>: Rimon Ghebremeskel, Tapi Goredema, Yi Ping, Elijah Smith, Samrawit Yemane</p>
<h2 id="news-our-dna-is-at-risk-of-hacking-warns-scientists">News: Our DNA is at risk of hacking, warns scientists</h2>
<p><strong><a href="/docs/Genomics-slides.pdf">Slides from Team 3</a></strong> (Lily Egenrieder, Mandy Le, Lara Mahajan, Kate McCray, Shreeja Tangutur)</p>
<p>Team 3 led a discussion about the growing involvement of AI in medicine, biology, and Genomics. They highlighted the benefits and risks of AI in these fields, one risk being vulnerability to the cybersecurity of Next Generation Sequencing (NGS) methods, which are more computer-based than their Sanger counterpart.</p>
<p>In early 2025, a growing number of scientists and technologists have begun sounding the alarm on a surprising new risk between the realms of biology and computing: the cybersecurity vulnerabilities of genetic data. As DNA sequencing becomes cheaper and faster, massive amounts of human genome data are now stored, shared, and analyzed across networks worldwide.</p>
<p>Researchers have discovered that these biological data streams can be targeted much like traditional digital systems, for example, by embedding malicious code into synthetic DNA or exploiting machine-learning models trained on genomic datasets. These findings raise urgent questions about privacy, ethics, and safety, especially as artificial intelligence becomes more deeply integrated into healthcare and biotechnology.</p>
<p><strong>Relevance to AI/topic of discussion</strong></p>
<p>With technology like AlphaFold and T7-Oracle advancing and integrating into fields of medicine and biology, conversations about AI’s impact and implications have increased. Genomics today isn’t just about biology; it has become deeply tied to computers and AI. Tools like AlphaFold, AI-based variant analysis, and platforms like T7-ORACLE are helping scientists interpret DNA, predict protein behaviour, and speed up evolution, demonstrating that AI can solve what used to take us years in a couple of days.</p>
<p>Despite the benefits, studies testing AI in medicine show that models can make different recommendations for identical patients based only on socioeconomic or demographic information. This is problematic as it can lead AI to choose a suboptimal diagnosis for a patient, disregarding medical necessity, and often choosing more advanced diagnostic tests for high-income patients while advising lower-income ones to undergo no further testing.</p>
<p>This is why AI bias and genomics belong in the same conversation. As genetic data becomes more digital and AI-driven, these biased decisions will pose a huge risk to patients, creating unfair outcomes if not unsafe/dangerous ones. If these models continue to be trained on incomplete and biased data, they will only be reinforcing existing inequalities in healthcare instead of fixing them with the new speed and computational power they provide.</p>
<center><img src="/images/class5-img1.webp" width=60% alt=""></img>
<p>Conceptual diagram of the T7-ORACLE accelerated protein evolution system in E. coli. (Source: <a href="https://www.nature.com/articles/s41589-025-02065-1">Nature</a>)</p>
</center> 
<h2 id="discussion">Discussion</h2>
<p>The discussion focused on how genomics has become more digital and what risks come with that change. The group explained that modern DNA sequencing methods, especially Next Generation Sequencing (NGS), rely heavily on computers, software, and online data storage. While this makes genetic research faster and more accessible, it also creates cybersecurity risks just like any other setting where untrusted parties may have control over input data. The slides showed that DNA data can be hacked or manipulated, including through synthetic DNA–encoded malware or AI-based attacks on genetic databases. Students also talked about how these risks connect to AI in healthcare.</p>
<h1 id="lead-discussion-protein-structure-prediction-alphafold">Lead Discussion: Protein Structure Prediction (AlphaFold)</h1>
<p><a href="/docs/AlphaFold-slides.pdf"><strong>Team 7 Slides</strong></a> (David Hu, Seth Lifland, Grace Kitthanawong, Pallavi Mamillapalli, Aryan Thodupunuri)</p>
<p>Readings:</p>
<ul>
<li>
<p>John Jumper, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, Russ Bates, Augustin Žídek, Anna Potapenko, Alex Bridgland, Clemens Meyer, Simon A. A. Kohl, Andrew J. Ballard, Andrew Cowie, Bernardino Romera-Paredes, Stanislav Nikolov, Rishub Jain, Jonas Adler, Trevor Back, Stig Petersen, David Reiman, Ellen Clancy, Michal Zielinski, Martin Steinegger, Michalina Pacholska, Tamas Berghammer, Sebastian Bodenstein, David Silver, Oriol Vinyals, Andrew W. Senior, Koray Kavukcuoglu, Pushmeet Kohli, and Demis Hassabis. <em>Highly accurate protein structure prediction with AlphaFold</em>. Nature, 26 August 2021. [<a href="/docs/alphafold2-jumper2021.pdf">PDF Link</a>]<br>
Optional: There is an &ldquo;AlphaFold 3&rdquo; now, described in <a href="/docs/alphafold3.pdf">this paper</a>, but you are not expected to read this one.</p>
</li>
<li>
<p>David Ampudia Vicente and George Richardson. <em>AI in Science: Emerging evidence from AlphaFold 2</em>. 25 November 2025. [<a href="/docs/ai_in_science_af2_igl_summary.pdf">PDF Link</a>] (Original paper and full report is available for <a href="https://www.innovationgrowthlab.org/resources/ai-in-science-alphafold-2">https://www.innovationgrowthlab.org/resources/ai-in-science-alphafold-2</a>, but you are only expected to ready the summary report in the <a href="/docs/ai_in_science_af2_igl_summary.pdf">PDF Link</a>.)</p>
</li>
</ul>
<center><img src="/images/class5-img2.webp" width=60% alt=""></img> 
<p>Figure 2: Protein structure produced by AlphaFold. (Source: <a href="https://alphafold.ebi.ac.uk/"><em>AlphaFold Protein Structure Database</em></a>)</p>
</center>
<p>The lead discussion for this class was presented by Team 7 regarding AlphaFold and Protein Structure Prediction. Team 7 discussed the potential benefits and concerns of AlphaFold’s involvement in protein research.</p>
<p>AlphaFold’s most impressive feat is its ability to generate remarkably accurate structure predictions of unreleased protein sequences. The paper reports on AlphaFold 2&rsquo;s success on the CASP14 competition, which is a blind test containing new experimentally solved and undisclosed structures, to which researchers/scientists can submit their predictions and gauge their accuracy. Unlike other computational methods, which can be complicated, expensive, or incorrect, AlphaFold was able to predict novel and large protein sequence structures with near experimental accuracy most of the time and often within the width of an atom, showing that the model can not only memorize, but also learn and extrapolate from given data.</p>
<p>AlphaFold cannot guarantee prefect accuracy, so empirical work is still needed to verify predicted structures when predicting novel protein sequences. Despite this, scientists can use AlphaFold’s prediction in conjunction with other computational methods, such as the evolutionary and physics-based models, to drastically reduce the time it takes to determine a protein’s 3D structure. The article also highlighted AlphaFold’s confidence score, which is usually correct for sections of the structure, providing a strong and realistic model for scientists to map from.</p>
<p>AlphaFold has already had substantial practical impact. In 2021, AlphaFold 2 was able to predict the Vg protein found in honeybees, which can help shorten the breeding cycle from a year to a couple of weeks, slowing down the extinction of honeybees. It also has involvement in sustainability efforts. Scientists were able to replicate the GLYK enzyme found in plants, an important enzyme for photosynthesis, without the same temperature weakness.</p>
<p>AlphaFold&rsquo;s ability to speed up protein structure research raises broader questions about AI autonomy in medicine and whether AI physicians can replace human ones. AlphaFold’s ability to extrapolate and generalize original solutions shows that AI could be able to autonomously solve critical and novel problems without needing a direct precedent, which is a huge advantage in the medical field.</p>
<h2 id="discussion-1">Discussion</h2>
<p>Team 7 mentioned many topics of discussion revolving around the use of AlphaFold and the future of AI in scientific research. Some questions were centered on conversations between table group members, but the larger discussion questions and their respective student responses became more about the broader implications of AlphaFold in conducting, as Team 7 stated, “good science.”</p>
<p><strong>Question 1</strong>: <em>Does AI change what it means to “do science”?</em></p>
<p>Team 7 argues that AI does change what it means by shifting the scope of problems that can be solved. One student responded to this question by stating that AI does not change what it fundamentally means to do science, but rather changes the process in which we do it. As they put it, “the scientific method does not change,” however, the tools that we use to pursue scientific research evolve over time. AI is simply just another tool in the toolbelt.</p>
<p>The discussion team then proposed a follow-up question for the class to ponder: Do we believe that AI will get to the point where it will perform new discoveries on its own, or will it follow human direction? Additionally, Professor Evans pointed out that AI is now being used to generate hypotheses for scientists to use in driving research. How will what it means to be a scientist change when much of the research agenda is driven by AI tools?</p>
<p><strong>Question 2</strong>: <em>Does AI narrow what “good science” looks like?</em></p>
<p>Team 7 opened this discussion with the authors’ findings in the AlphaFold 2 Summary Report. The authors stated that AlphaFold 2 “lowers the cost barrier and reduces the risks associated with research portfolio diversification,” which may help divert research efforts and resources to more unexplored areas of protein science. However, the discussion team also mentioned how there is discourse as to whether AI will either contribute to or help mitigate the “spotlight effect,” which is a phenomenon where research is focused on existing, data-rich problems and structures.</p>
<p>Some students shared their disagreement with the discussion question. One student asserted that since AI programs like AlphaFold 2 are tools for scientists to use, it is more of the scientists’ responsibility to know how to narrow their focus. AlphaFold 2 cannot perform research on its own. A member of Team 7 shared the perspective that even if AI could narrow the scope of pursued questions, it is still good to solve problems in any case, no matter how easy and well-established they may be.</p>
<p><strong>Question 3</strong>: <em>If AlphaFold is getting protein structures right most of the time, does it still make sense to run expensive lab experiments to confirm them? And is it okay if we don’t fully understand how the model works, as long as the output is correct?</em></p>
<p>Team 7 led with the idea that “high predictive accuracy does not necessarily imply mechanistic understanding”. In essence, AlphaFold 2, along with many other AI systems, works as a “black box” where most users only acknowledge that its protein structure predictions are accurate; however, they do not understand how the conclusion was reached. In light of this, should researchers place so much emphasis on the results of AlphaFold 2?</p>
<p>One student’s response was a more nuanced take. The student affirmed that we should not take an AI’s word at face value, especially when conducting experiments. On the other hand, researchers do not necessarily need to know the nitty-gritty of how the model works. If the model proves useful, then it should be recognized as such, but it does not discount the importance of verification through experimentation.</p>
<p>Overall, although the implications of AI systems such as AlphaFold hold a strong potential to reshape how science is conducted in the future, the class seemed to hold the idea that the current process of pursuing science is in no danger of change. Optimistically, AI can divert resources to open entirely new fields of research to advance the scope of our knowledge. Only time will tell where it can push us next.</p>

      </div>

      <meta itemprop="wordCount" content="1718">
      <meta itemprop="datePublished" content="2026-01-29">
      <meta itemprop="url" content="http://localhost:1313/blogs/class5/">
    </article>
</div>


    </main>
    
    
<footer class="whatisthis">
  <hr />
  <div class="row">
    <div class="column small-12 medium-6">
      <a href="//aihumanityclass.github.io"><b>cs 4501: humAnIty</b></a><br>
      <a href="//www.cs.virginia.edu">University of Virginia</a>
    </div>
    <div class="column small-14 medium-5">
      <font size="-1">
      Subscribe to
	the <a href="/index.xml"><i class="fa fa-rss-square"></i>&nbsp;RSS feed</a>.
      <a id="searchsite">
	<form method="get" action="https://duckduckgo.com/">
	  <label for="search-field" class="show-for-sr">Search with DuckDuckGo</label>
	  <input type="search" name="q" maxlength="255" placeholder="Search with DuckDuckGo" id="search-field">
	  <input type="hidden" name="sites" value="http://localhost:1313/"/>
	  <input type="hidden" name="k7" value="#faf8f8"/>
	  <input type="hidden" name="kj" value="#b33"/>
	  <input type="hidden" name="ky" value="#fafafa"/>
	  <input type="hidden" name="kx" value="b"/>
	  <input type="hidden" name="ko" value="-1"/>
	  <input type="hidden" name="k1" value="-1"/>
	  <input type="submit" value="DuckDuckGo Search" style="visibility: hidden;" />
	</form>
      </a>
</font>
    </div>
  </div>
  </hr>
</footer>


    
    <div class="endofpage">
    </div>

    <script src="/js/jquery.js"></script>
    <script src="/js/what-input.js"></script>
    <script src="/js/foundation.min.js"></script>
    <script src="/js/finite.js"></script>

    
    <script src="/js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>

    
    
    
  </body>
</html>
